decompiler/extractor/main.cpp

```diff
@@ -313,6 +325,17 @@ void compile(std::filesystem::path extracted_iso_path) {
   compiler.make_system().set_constant("*iso-data*", absolute(extracted_iso_path).string());
   compiler.make_system().set_constant("*use-iso-data-path*", true);
 
+  auto buildinfo_path = (extracted_iso_path / "buildinfo.json").string();
+  auto bi = parse_commented_json(file_util::read_text_file(buildinfo_path), buildinfo_path);
+  auto all_flags = bi.at("flags").get<std::vector<std::string>>();
+  int flags = 0;
+  for (const auto& n : all_flags) {
+    if (auto it = sGameIsoFlagNames.find(n); it != sGameIsoFlagNames.end()) {
+      flags |= it->second;
+    }
+  }
+  compiler.make_system().set_constant("*jak1-full-game*", flags & FLAG_JAK1_BLACK_LABEL);
```

Comment:
isn't this swapped? If we have black label, we don't have jak1-full-game?

Reply:
oops